apps/judicial-system/web/pages/_document.tsx

```diff
@@ -10,8 +10,6 @@ export default class MyDocument extends Document {
           <div id="main">
             <Main />
           </div>
-          {/* Here we will mount our modal portal */}
```

Comment:
FYI: I'm moving the modal away from here because this is rendered server side and is therefore inaccessible to our tests.